manage.py

```diff
@@ -12,6 +22,18 @@ def cli():
 @cli.command()
 def init():
     Base.metadata.create_all(engine)
+    User._bootstrap()
+    sess = Session()
+    for user in sess.query(User).all():
+        Account._bootstrap(user)
```

Comment:
You're calling psuedo-private static methods directly from the class. This is bad practice. Python technically doesn't have private methods, but an underscore marks as method for internal use within the module/class.
https://pep8.org/#descriptive-naming-styles